### PR TITLE
[Metrics-generator] Add section about exporting metrics

### PR DESCRIPTION
**Pull request by mapno**
**What this PR does**:

Briefly explains how metrics are exported from the metrics-generator to the metrics backend.

### CODE REVIEW

File: docs/sources/tempo/metrics-generator/_index.md [hunk 1]
```diff
@@ -46,3 +46,9 @@ Dimensions can be the service name, the operation, the span kind, the status cod
 The more dimensions are enabled, the higher the cardinality of the generated metrics.
 
 To learn more about this processor, read the [documentation]({{< relref "span_metrics/" >}}).
+
+### Exporting metrics
+
+The metrics-generator runs a Prometheus Agent that periodically sends metrics to a remote_write endpoint.
+The remote_write endpoint is configurable and can be any Prometheus-compatible endpoint.
+In order to support multi-tenancy, the metrics-generator forwards the `X-Scope-OrgID` header to the remote_write endpoint.
```
Comment (knylander-grafana) [inline docs/sources/tempo/metrics-generator/_index.md:54]: I added links to the configuration file and to the Prometheus read_write endpoint (which is linked in the Config file).

```suggestion
The metrics-generator runs a Prometheus Agent that periodically sends metrics to a `remote_write` endpoint.
The `remote_write` endpoint is configurable and can be any [Prometheus-compatible endpoint](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
To learn more about the endpoint configuration, refer to the [Metrics-generator]({{< relref "../configuration/#metrics-generator" >}}) section of the Tempo Configuration documentation. 

To support multi-tenancy, the metrics-generator forwards the `X-Scope-OrgID` header to the remote_write endpoint.
```

**Review by knylander-grafana — APPROVED** [review]
Thank you for adding doc! I have one suggestion that adds links to relevant documentation.

File: docs/sources/tempo/metrics-generator/_index.md [hunk 2]
```diff
@@ -46,3 +46,9 @@ Dimensions can be the service name, the operation, the span kind, the status cod
 The more dimensions are enabled, the higher the cardinality of the generated metrics.
 
 To learn more about this processor, read the [documentation]({{< relref "span_metrics/" >}}).
+
+### Exporting metrics
```
Comment (zalegrala) [inline docs/sources/tempo/metrics-generator/_index.md:50]: Can we mention the internal that we use to send is configurable also?

File: docs/sources/tempo/metrics-generator/_index.md [hunk 3]
```diff
@@ -46,3 +46,11 @@ Dimensions can be the service name, the operation, the span kind, the status cod
...
```
Comment (kvrhdn) [inline docs/sources/tempo/metrics-generator/_index.md:50]: What do you think about renaming this to "remote writing metrics"? Exporting makes it seem as if they are stored long-term in the metrics-generator.

I don't feel particularly strongly about this though.

File: docs/sources/tempo/metrics-generator/_index.md [hunk 4]
```diff
@@ -46,3 +46,11 @@ Dimensions can be the service name, the operation, the span kind, the status cod
...
+
+The metrics-generator runs a Prometheus Agent that periodically sends metrics to a `remote_write` endpoint.
+The `remote_write` endpoint is configurable and can be any [Prometheus-compatible endpoint](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
+To learn more about the endpoint configuration, refer to the [Metrics-generator]({{< relref "../configuration/#metrics-generator" >}}) section of the Tempo Configuration documentation. 
+
+To support multi-tenancy, the metrics-generator forwards the `X-Scope-OrgID` header to the remote_write endpoint.
```
Comment (kvrhdn) [inline docs/sources/tempo/metrics-generator/_index.md:56]: Small nit: we only do this when multitenancy is enabled
```suggestion
When multi-tenancy is enabled, the metrics-generator forwards the `X-Scope-OrgID` header of the original request to the remote_write endpoint.
```

File: docs/sources/tempo/metrics-generator/_index.md (hunk 2 above)
Comment (zalegrala) [reply docs/sources/tempo/metrics-generator/_index.md:50]: Sorry typo.  The `interval` is what I meant, like how often we write metrics.

File: docs/sources/tempo/metrics-generator/_index.md (hunk 3 above)
Comment (mapno) [reply docs/sources/tempo/metrics-generator/_index.md]: Done!

File: docs/sources/tempo/metrics-generator/_index.md (hunk 2 above)
Comment (mapno) [reply docs/sources/tempo/metrics-generator/_index.md]: Done!

**Pull request merged**